CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+1.4.0 / 2019-08-28
+==================
+
+###### Changes
+* Dropping support for PHP 5.3
+* Updated documentation about testing
+* Syn library is [0.0.2](https://github.com/bitovi/syn/tree/v0.0.2)
+* `setDesiredCapabilities` will throw an exception if you will try to set capabilities on already started session
```

Comment:
Please move into `Misc` sub-section and replace with:

```
The `setDesiredCapabilities` method will throw an exception, when used on a started session
```

Reply:
moved